model/http-common.yaml

```diff
@@ -52,6 +52,13 @@ groups:
         requirement_level: required
         brief: >
           Port identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to.
+      - ref: http.route
+        requirement_level: opt_in
+        brief: >
+            The low-cardinality path template associated with outgoing HTTP request.
```

Comment:
maybe make clear that "path" here can mean both path and query string?

Reply:
technically this attribute could contain not just path, but path and query and also path,query and fragment, as long as the low-cardinality requirement is met. Should we perhaps refer to it as an **absolute path reference**, as defined in https://www.rfc-editor.org/rfc/rfc3986#section-4.2 ?

Reply:
I don't see "absolute path reference" on that page?

additionally could include an example that uses query string

Reply:
> I don't see "absolute path reference" on that page?

See section 4.2 of the RFC: " A relative reference that begins with a single slash character is termed an absolute-path reference."